modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -10,21 +10,87 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-When you deploy an application to CloudHub, you have access to the validated OS updates, security patches, and bug fixes that MuleSoft provides regularly.
 
-You can keep your CloudHub applications up to date by manually updating them at your convenience.
-If you don't update apps manually, MuleSoft applies the updates automatically.
+To ensure the security and stability of CloudHub, MuleSoft releases security updates
+and monthly patches and applies them automatically on a schedule.
 
-When a new runtime version is available for your CloudHub application, the *Applications* page displays a banner and the *Update Available* tab lists the apps that require updates:
+Automatic updates use blue-green deployment, which ensures zero downtime.
 
+== Security Updates 
+
+MuleSoft regularly runs scans to identify security vulnerabilities in Mule runtime engine, JVM,
+and the underlying operating system, and then automatically applies security patches based on the following Salesforce SLA:
+
+[%header,cols="20,25,40"]
+|===
+|Severity Level | Severity Definition | Patch Applied Within 
+|P0 | Critical | 7 days
+|P1 | High | 30 days
+|===
+
+To minimize disruption, the monthly date patch update always includes patches for any P1 security issues.
```

Comment:
```suggestion
To minimize disruption, the monthly date patch update always includes patches for any P1 security issues.
```
please remove this statement. We can add it in the patching section where the date patch has been introduced